cylc/flow/task_pool.py

```diff
@@ -1123,7 +1123,11 @@ def _reload_taskdefs(self) -> None:
         # Log tasks orphaned by a reload but not currently in the task pool.
         for name in orphans:
             if name not in (itask.tdef.name for itask in tasks):
-                LOG.warning("Removed task: '%s'", name)
```

Comment:
Agreed.

The user specifically got Cylc to remove tasks from the workflow, why would we warn them about this.